src/views/AccountDetail.vue

```diff
@@ -11,7 +11,7 @@
         <template #error>Account {{address}} is not exist</template>
       </Card>
 
-      <Card v-if="1||showMultiSigDeatail" class="card-f card-adaptive" :loading="loading">
```

Comment:
i think `showMultiSigDeatail` is valild because it existing in `computed` , only `1||` have to remove.

Reply:
Referring the issue #134 all this cards should be shown even there is no items in a list. It is the reason I removed `v-for`s. I havent removed computed props linked with that, so we can go back to this`v-for` approach if needed

Reply:
i see, got it